iceoryx_posh/source/roudi/application/roudi_app.cpp

```diff
@@ -12,8 +12,6 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-//
-// SPDX-License-Identifier: Apache-2.0
```

Comment:
@chiranjeevimaddi Can you revert this file ?

Reply:
Done